README.rst

```diff
@@ -16,15 +16,11 @@ support at `google._async_resumable_media`.
 
 Supported Python Versions
 -------------------------
-Python >= 3.5
-
-Deprecated Python Versions
---------------------------
-Python == 2.7. Python 2.7 support will be removed on January 1, 2020.
```

Comment:
Ditto here, AFAIK we should retain the "unsupported Python versions" sections and explain what the last backward-compatible library version is.

Reply:
87c3d72